<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectSpectrumSelector</class>
 <widget class="QWidget" name="IndirectSpectrumSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QLabel" name="lbSelectionMode">
     <property name="text">
      <string>Fit Spectra</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbSelectionMode">
     <item>
      <property name="text">
       <string>Range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>String</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbColon">
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="swSpectraSelection">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loSpectraRange">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spMinimumSpectrum"/>
       </item>
       <item>
        <widget class="QLabel" name="lbSpectraRangeTo">
         <property name="text">
          <string>to</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spMaximumSpectrum"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lpSpectraText">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="leSpectra"/>
       </item>
       <item>
        <widget class="QLabel" name="lbSpectraError">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lbMaskSpectrum">
     <property name="text">
      <string>Mask Bins of Spectrum</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="swMaskSpectrumSelection">
     <widget class="QWidget" name="loMaskSpectrumSp">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QSpinBox" name="spMaskSpectrum"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loMaskSpectrumCb">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="cbMaskSpectrum"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMaskBins"/>
   </item>
   <item>
    <widget class="QLabel" name="lbMaskBinsError">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbSelectionMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>swSpectraSelection</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSelectionMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>swMaskSpectrumSelection</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
